### DISCUSSION TITLE
Change priority of customTags in autocomplete suggestion

### DISCUSSION THREAD

**Discussion by LavaToaster** [redhat-developer/vscode-yaml#739] [Q&A]
👋  Hey Folks!

I'm using a combination of the AWS Toolkit, Azure Pipelines and VSCode Yaml plugins in my vscode setup. 

I'm facing an issue where the AWS Toolkit mutates my user settings to add a `yaml.customTags` config option (or append to it if one exists). These tags have priority over any intelligent language server like the Azure Pipeline ones and result in some interesting ordering.

![CleanShot 2022-04-03 at 23 38 54](https://user-images.githubusercontent.com/838613/161451869-273fc904-8d3b-4799-bc7f-14a7c3751661.png)

Is there any way I can de-prioritize the weight of custom tags for their completion/IntelliSense order?

**Comment by gorkem** [comment]
We do not specify a weight for the completion items from this extension. I believe the order is just the alphabetical order, the `!` character just groups them on top.

**Comment by bombillazo** [comment]
Dang, I was looking for this as well, Ideally custom schema props have higher priority than those !X suggestion, much like TypeScript shows props first...

**Comment by ghost** [comment]
Martyb166
